docs/guides/deployment.rst

```diff
@@ -1,27 +1,64 @@
 Deployment
 ==========
 
-Docker
-~~~~~~
+One line deploy
+~~~~~~~~~~~~~~~
 
-We continuously deploy our `Hoodie Tracker App`_ using Docker. You can read
-about our continuous
-deployment set at `hoodie-app-tracker/deployment.md`_.
+After you’ve built your Hoodie app you probably want to put it online. You can choose to deploy your app as read-only or deploy the backend couchdb database as well. This `video`_ and the text below descibes how to deploy your app using one line of code. Alternatively, you can deploy your app using Docker, please refer to the Docker section. 
 
-.. _Hoodie Tracker App: https://github.com/hoodiehq/hoodie-app-tracker
-.. _hoodie-app-tracker/deployment.md: https://github.com/hoodiehq/hoodie-app-tracker/blob/maste/deployment.md
+.. _video: https://youtu.be/29Uclxq_1Vw
+Deploying to Now
+~~~~~~~~~~~~~~~~
+.. _command line tool: https://github.com/zeit/now-cli
+`Now`_ allows you to deploy a Node application with its `command line tool`_. It’s 100% free for Open Source projects. You can deploy an app from your computer or right from a GitHub repository. For example, to deploy our Hoodie Tracker demo app all you have to do is to run this command:
+::
+
+$ now hoodiehq/hoodie-app-tracker --npm -e NODE_ENV=production -e hoodie_inMemory=true,
 
-Now
-~~~
+To decribe this further:
 
-`now`_ is a great way to quickly deploy Node.js applications.
-Unfortunately, now is a read-only file system, so you must either run
-your app in memory or set an external CouchDB URL.
+- hoodiehq/hoodie-app-tracker, is the GitHub repository slug.
 
-Add this script to your package.json and you are good to go:
+- --npm tells now to deploy using npm, as there is also Dockerfile in the repository.
+
+- -e NODE_ENV=production sets the NODE_ENV environment variable to production, which makes the deployment faster as no devDependencies will be installed. 
+
+- -e hoodie_inMemory=true makes the Hoodie app run in-memory mode, meaning that no data is persisted and no files are written. This is important because now is a read-only file system. That means that all user accounts and data will be lost on the next deployment, but it is great for for a quick test or demo of your application.
```

Comment:
What do you think about doing inline code highlights for `hoodie/hoodie-app-tracker`, `--npm`, `-e NODE_ENV=production` and `-e hoodie_inMemory=true`? You can do it with 

```
:code:`highlight here`
```